version.php

```diff
@@ -29,15 +29,15 @@
 // between betas, final and RCs. This is _not_ the public version number. Reset minor/patchlevel
 // when updating major/minor version number.
 
-$OC_Version = array(18, 0, 0, 8);
+$OC_Version = array(19, 0, 0, 0);
 
 // The human readable string
-$OC_VersionString = '18.0.0 RC1';
+$OC_VersionString = '18.0.0 alpha';
```

Comment:
```suggestion
$OC_VersionString = '19.0.0 alpha';
```
Shouldn't this be 19 alpha?

Reply:
nah we always lower cased it...

Reply:
What about the 19 part :p

Reply:
Oh okay, just got the impression from here
https://github.com/nextcloud/server/blob/04ef434af079e22ef27a874975970fde4ae93961/version.php#L35

Also, still need to bump the version number in the name.

Reply:
ah hehe fair enough but CI also fails. let me fix it all